jest.setup.ts

```diff
@@ -0,0 +1,4 @@
+import '@testing-library/jest-dom/extend-expect';
+
+setupFilesAfterEnv: ['<rootDir>/jest.setup.ts']
```

Comment:
remove this line from here and put it into the `jest.config.js` file like so:
![image](https://github.com/Alex-2kZharkov/oldboy-barbershop/assets/66321831/a347c20e-199e-4090-a142-ac61da98eb45)


Thus you will include `@testing-library/jest-dom/extend-expect` for each test automatically